src/citrine/resources/delete.py

```diff
@@ -71,31 +72,38 @@ def _async_gemd_batch_delete(
 
     if dataset_id is not None:
         body.update({'dataset_id': str(dataset_id)})
-
-    path = format_escaped_url('/projects/{project_id}/gemd/async-batch-delete',
-                              project_id=project_id
+    if team_id != None:
```

Comment:
And `is not None`

Reply:
Fixed on 8d39c24072b10d099c8a0eaf580aa198cbb8efcf